cluster/manifests/roles/cockpit-skipper-rbac.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
@tkrop We need to roll this out separately to ensure that we don't drop access to `cockpit-skipper` during the rollout.

Reply:
They currently do not even have access. This request was issued to reflect the roll out #2252. So if something breaks we do not care here ;).

Reply:
ok